packages/suite/src/config/wallet/networks.ts

```diff
@@ -232,7 +232,7 @@ const networks = {
             tx: 'https://tbtc1.trezor.io/tx/',
             account: 'https://tbtc1.trezor.io/xpub/',
         },
-        features: ['rbf', 'sign-verify'],
+        features: ['rbf', 'sign-verify', 'satoshis'],
```

Comment:
```suggestion
        features: ['rbf', 'sign-verify', 'amount-unit'],
```
let's keep consistent naming